docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/requests-module.mdx

```diff
@@ -4,42 +4,42 @@ sidebar_order: 2000
 description: "Learn how to manually instrument your code to use Sentry's Requests module."
 ---
 
-As a prerequisite to setting up [Requests](/product/performance/requests/), you’ll need to first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink>. Once this is done, the JavaScript SDK will automatically instrument outgoing HTTP requests. If that doesn't fit your use case, you can set up using [custom instrumentation](#custom-instrumentation).
+As a prerequisite to setting up [Requests](/product/performance/requests/), you'll need to first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink>. Once this is done, the JavaScript SDK will automatically instrument outgoing HTTP requests. If that doesn't fit your use case, follow this guide to manually instrument your requests.
 
 ## Custom Instrumentation
 
```

Comment:
```suggestion
```

No longer needed, since the only content of this page is how to custom instrument requests!

Reply:
Good catch, thanks!